.gitignore

```diff
@@ -9,6 +9,7 @@
 # dependencies
 /node_modules
 yarn.lock
+/tools/schematics/node_modules/
```

Comment:
The schematics should not be installed within the tools folder, I guess. 
Can you please remove this line?

Reply:
Was it something wrong on my side that caused it to get installed there?

Reply:
I don't really know why this was installed here, tbh. 
Thank you for cleaning it up though.

Reply:
Might be the intellij npm install then, It was fabian who removed it in the end :)